CHANGES_NEXT_RELEASE

```diff
@@ -11,4 +11,4 @@
 - Fix: GET /v2/entities/id/attrs/name/value for type date (Issue #1885)
 - Fix: PUT /v2/entities/id/attrs/name/value forbidden chars (JSON) (Issue #1905)
 - Fix: PUT /v2/entities/id/attrs/name/value changing type to 'none' (text/plain) (Issue #1904)
-
+- Add: PUT /admin/log?level=XXX to change log level in run-time (Issue #1913)
```

Comment:
Not sure that is a good idea ...
Everything in the list would be initial, and the fact that it can be changed in run-time has nothinf to do with the CLI params ...
If you really want it in the 'usage', I would put it like this:

    [option '-logLevel' <log level (NONE, ERROR, WARNING, INFO, DEBUG)> (configurable in run-time)]
￼
Or something like that ...